### PR TITLE
Feat : Home Page

### PR DESCRIPTION
**Pull request by vikrantkode**
# Open for Reviews
## Home Page (Deploy preview not available)

Screenshot (111)

Screenshot (112)

### CODE REVIEW

**Review by harshitpaliwal95 — APPROVED** [review]
Looks good

File: CSS/header.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    color: white;
+}
+
+.btn-primary:hover{
+    color: white;
+}
+
+.btn-primary{
+    margin: 0;
+    background-color: white;
```
Comment (raieegulhane) [inline CSS/header.css]: Use variable for color. Even white.

**Review by raieegulhane — COMMENTED** [review]
Check comment. Else good to merge.

**Comment by TJain02** [comment]
Good to go!

File: CSS/header.css (hunk 1 above)
Comment (vikrantkode) [reply CSS/header.css]: Completed

**Pull request merged**